CHANGES_NEXT_RELEASE

```diff
@@ -8,3 +8,4 @@ Fix: Changing cygnus port does not work (#1698)
 [cygnus-ngsi] Changed log level for some events (#1769)
 [cygnus-ngsi][PostgisSink, PostGRESQLSink] Support for native Postgres and Postgis types (#1780)
 [cygnus-ngsi][MySQL] Support for native MySQL types (#1780)
+[cygnus-ngsi] Add correlatorId rollbacked transactions (#1770)
```

Comment:
Mabye better the next one? (not 100% sure)

```suggestion
[cygnus-ngsi] All the retries are now using the same correlatorid (#1770)
```

Reply:
Not really sure about that @fgalan. All correlatorIds (inside the batch) are stored on a String divided by ",", so since I split that String every entry is supposed to have a different one. 

Do you agree?

Reply:
Complex to be explained/understood by comments :). Let's talk F2F or by skype.